app/models/project.rb

```diff
@@ -124,7 +124,6 @@ def initialize(retry_after_seconds = nil)
   delegate :code_of_conduct_url, :contribution_guidelines_url, :funding_urls, :security_policy_url, to: :repository, allow_nil: true
 
   validates :name, :platform, presence: true
-  validates :name, uniqueness: { scope: :platform, case_sensitive: true }
```

Comment:
cc @havocp since we've talked about redundant AR validations + UNIQUE indices in the past. I personally love the idea of cleaning up these redundancies whenever we see them